datajoint/user_tables.py

```diff
@@ -152,6 +157,75 @@ class Lookup(UserTable):
     )
 
 
+class Params(UserTable):
+    """
+    Inherit from this class if the table's values are paramsets. Adds hashing of fields
+    that include 'param' and stores as additional unique index 'paramset hash'
+    """
+
+    _prefix = "##"
+    tier_regexp = (
+        r"(?P<params>" + _prefix + _base_regexp.replace("TIER", "params") + ")"
+    )
+
+    def __init__(self):
+        super().__init__()
+        if "params" not in self.definition:
```

Comment:
What does this condition test?  This would return `True` if the definition contains the substring `"params"` anywhere in the definition. This is not a meaningful test.